website/cue/reference/components.cue

```diff
@@ -210,9 +210,10 @@ components: {
 				enabled: true
 			}
 
-			// `healtcheck` notes if a component offers a healthcheck on boot.
```

Comment:
🤦

Reply:
Beat you to a spelling error for once! :D